src/main/java/io/confluent/connect/elasticsearch/Validator.java

```diff
@@ -33,6 +33,7 @@
 import static io.confluent.connect.elasticsearch.ElasticsearchSinkConnectorConfig.SECURITY_PROTOCOL_CONFIG;
 import static io.confluent.connect.elasticsearch.ElasticsearchSinkConnectorConfig.SSL_CONFIG_PREFIX;
 
+import io.confluent.connect.elasticsearch.ElasticsearchSinkConnectorConfig.SecurityProtocol;
```

Comment:
nit: usually static imports will follow regular imports. 
Not sure we need more stylistic changes though. Just noticed.